bed2interval_list.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+module load bioinfo-tools picard
+java -jar ${PICARD_HOME}/picard.jar BedToIntervalList \
```

Comment:
Noted that this is just an example script but maybe it would be good to clarify this with an inline comment, in particular that the file names refer to hg19 (perhaps mostly this was a bit unexpected since the other scripts assume GRCh38..). I would perhaps suggest also to rename the file to e.g. `bed2interval_list.example.sh`